### ISSUE TITLE
week_calendar time zone issue

### ISSUE THREAD

**Issue by mosinski** [excid3/simple_calendar#137]
Hi,
I noticed recently that my application in standard generated week calendar is used UTC time and not time zone from **application.rb**
```application.rb
    config.time_zone = 'Warsaw'
    config.beginning_of_week = :monday
```
Do I have to declare my time zone in some other way?

gem version:
```Gemfile
Using simple_calendar 2.1.2
```
Is this file: [calendar.rb](https://github.com/excid3/simple_calendar/blob/master/lib/simple_calendar/calendar.rb#L77)
instead **Date.today** should be **Date.current** :grey_question:

**Comment by excid3** [comment]
That's probably it. Date.today doesn't respect the timezone I believe. I'm also using `Time.zone.now.to_date` on line 32. In both cases it probably makes more sense to use `Date.current`. 

Want to fork this and modify those two lines to use `Date.current` and see if that fixes your problem? If it does then we can merge in that and release a new version.

Good catch! 👍

**Comment by defis** [comment]
I spent my whole life to solve this problem, thanks a lot mosinski :O

**Comment by jgalka** [comment]
good job mosinski :)

**Comment by mosinski** [comment]
Thx :smiley:
In my case, this two lines fixes the problem :)

**Comment by excid3** [comment]
Fixed in #138 and released a new version for ya.
